### DISCUSSION TITLE
2D image output format PDF

### DISCUSSION THREAD

**Discussion by xubaozi1307** [neperfepx/neper#596] [Q&A] [answered]
Hello Professor! I want to import the 2D image generated in neper to CAD, but directly convert the .png to .dxf file is very poor, is there any way to convert the real space image to vectorial PDF format? 
I used `imageformat pdf`,but it seems to be useful only under the pole figure.
Or is there any other way to export 2d image as vectorial file?

Thank you very much for your excellent work and look forward to your reply！

**Comment by rquey** [comment, marked as answer]
True that module -V generates only PNG images for standard visualizations (real space). See maybe the different [-format](https://neper.info/doc/neper_t.html?highlight=format#cmdoption-format) possibilities in -T.
